### ISSUE TITLE
[Uploader] Preload File error when empty array

### ISSUE THREAD

**Issue by coolhome** [syncfusion/ej2-javascript-ui-controls#47]
## Problem:
If the property array `files` is empty the following error is thrown.

```
vue.esm.js:628 [Vue warn]: Error in updated hook: "TypeError: Cannot read property 'size' of undefined"
```

Source File: https://github.com/syncfusion/ej2-javascript-ui-controls/blob/be01d3d68247da5a06d7ccef1b8e12dbfa9eac51/controls/inputs/src/uploader/uploader.ts#L1177

Browsing through the source I believe this empty element is for an uploaded file before a user selects one.

Minimal Vue Example:
```vue
<ejs-uploader :files='[]' />
```

## Use Case:
When a user returns back to the page it would be useful to show the user what was uploaded.

```vue
<ejs-uploader>
    <e-files>
        <e-uploadedfiles 
            v-for="file of myFiles" 
            :key='file.name' 
            :name='file.name' 
            :type='file.type' 
            :size='file.size' />
    </e-files>
</ejs-uploader> 
```

## Workaround
Add empty object  or one `e-uploadedFiles` element for the `files` property.

```vue
<ejs-uploader :files='[{}]' />
```

**Comment by BerlyChristopher** [comment]
Hi Preston Alvarado,

Greetings from Syncfusion.

We have confirmed the reported issue as a bug at our end and this fix will be included in our patch release scheduled on 5th June 2019. You can track the status of your request, review the proposed resolution timeline, and contact us for any further inquiries through the below feedback link. Please log in to the feedback portal tool using Direct trac credentials. We appreciate your patience until then. 

**Feedback Link:** https://www.syncfusion.com/feedback/6486/if-files-are-empty-array-it-throws-an-console-error

Regards,
Berly B.C

**Comment by PrinceOliver** [comment]
Hi @coolhome ,

Thank you for your patience.

Hi Team,

We have fixed the reported issue and the fix is included in the version **17.1.50**. We suggest you upgrade to the latest version. Please let us know if you need any further assistance on this.

Regards,
Prince


Regards,
Prince
